api/spec/src/subscription.tsp

```diff
@@ -0,0 +1,266 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi3";
+
+import "./types.tsp";
+import "./ratecards.tsp";
+import "./plan.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.OpenAPI;
+
+namespace OpenMeter;
+
+@route("/api/v1/subscriptions")
+@tag("Subscriptions")
+interface Subscriptions {
+  @get
+  @operationId("getSubscription")
+  get(@path subscriptionId: ULID): Subscription | OpenMeter.NotFoundError | OpenMeter.CommonErrors;
+
+  create(
+    @body body: {
+      customerId: ULID;
+      templatingPlan: {
```

Comment:
Is `templatingPlan` an industry-standard phrase? It sounds weird to me.

Reply:
they usually call it plan

Reply:
In case I wasn't clear, the `templating` part is what's weird to me. Something like `basePlan` sounds more natural, but I'm not familiar with the lingo.

Reply:
+1.